web/concrete/src/Application/Application.php

```diff
@@ -130,6 +130,45 @@ public function clearCaches()
         OpCache::clear();
     }
 
+   /**
+     * Utility method for clearing page caches and block caches that has btCacheBlockOutputClearOnEvents option enabled
+     */
+    public function partialClearCaches()
+    {
+        \Events::dispatch('on_partial_cache_flush');
```

Comment:
Yes i know. the method that i suggest is, instead of calling 
<code>
$app = Application::getFacadeApplication();
$db = $app['database']->connection();
</code>
we can use a wrapper method for that
<code>
...
class Application extends Facade{
...
public static function getDatabase(){
      $cms = static::getFacadeApplication();
      return $app['database']->connection();
}
...
</code>